pkgs/top-level/aliases.nix

```diff
@@ -1143,6 +1143,7 @@ mapAliases ({
   squid4 = squid;  # added 2019-08-22
   sshfsFuse = throw "'sshfsFuse' has been renamed to/replaced by 'sshfs-fuse'"; # Converted to throw 2022-02-22
   stanchion = throw "Stanchion was part of riak-cs which is not maintained anymore"; # added 2020-10-14
+  steam-run-native = steam-run; # added 2022-02-21
```

Comment:
I think it would be better to make this into a throw since the package was already broken

Reply:
This is what I had originally, but I don't feel strongly either way.

Reply:
I don't feel super strongly either way, but it would probably be a better user experience to have an alias